gwml2_schema.sql

```diff
@@ -0,0 +1,102 @@
+CREATE SCHEMA groundwater;
+
+CREATE TABLE groundwater.CI_Telephone (
+	id serial PRIMARY KEY,
+	voice VARCHAR(50),
+	facsimile VARCHAR(50)
+);
+
+CREATE TABLE groundwater.CI_Address (
+	id serial PRIMARY KEY,
+	deliveryPoint VARCHAR(50),
+	city VARCHAR(150),
+	administrativeArea VARCHAR(150),
+	postalCode VARCHAR(50),
+	country VARCHAR(150),
+	electronicMailAddress VARCHAR(150)
+);
+
+CREATE TABLE groundwater.CI_OnlineResource (
+	id serial PRIMARY KEY,
+	linkage VARCHAR(150),
+	protocol VARCHAR(150),
+	applicationProfile VARCHAR(250),
+	name VARCHAR(150),
+	description VARCHAR(255),
+	function groundwater.CI_OnLineFunctionTerm
+);
+
+CREATE TABLE groundwater.CI_Contact (
+	id serial PRIMARY KEY,
+	phone int4 REFERENCES groundwater.CI_Telephone(id) ON DELETE SET NULL,
+	address int4 REFERENCES groundwater.CI_Address(id) ON DELETE SET NULL,
+	onlineResource int4 REFERENCES groundwater.CI_OnlineResource(id) ON DELETE SET NULL,
+	hoursOfService VARCHAR (255),
+	contactInstructions VARCHAR (255)
+);
+
+CREATE TABLE groundwater.CI_ResponsibleParty (
+	id serial PRIMARY KEY,
+	individualName VARCHAR (255),
+	organisationName VARCHAR (255),
+	positionName VARCHAR (255),
+	contactInfo int4 REFERENCES groundwater.CI_Contact(id) ON DELETE SET NULL,
+	role groundwater.CI_RoleTerm
+);
+
+CREATE TABLE groundwater.GM_Envelope (
+	id serial PRIMARY KEY,
+	upperCorner GEOMETRY,
+	lowerCorner GEOMETRY
+);
+
+CREATE TABLE groundwater.borehole (
+	id serial PRIMARY KEY,
+	bholeDateOfDrilling DATE,
+	bholeInclinationType groundwater.BoreholeInclinationTerm,
+	bholeMaterialCustodian int4 REFERENCES groundwater.CI_ResponsibleParty(id) ON DELETE SET NULL,
+	bholeNominalDiameter float(8),
+	bholeOperator int4 REFERENCES groundwater.CI_ResponsibleParty(id) ON DELETE SET NULL,
+	bholeStartPoint groundwater.BoreholeStartPointTypeTerm
```

Comment:
mistake? needs fkey ref and vocab table

Reply:
because i separate the vocabs in the gwml2_vocabularies.sql file

Reply:
The table definition should be here, the table data shoudl be in vocabs